src/components/Login/LoginContainer.tsx

```diff
@@ -191,6 +191,9 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
       try {
         const { key, endpoint } = decodeQr(qrCode);
         Vibration.vibrate(50);
+        const endpointCheck = process.env.DOMAIN_CHECK as string;
+        if (!RegExp(endpointCheck).test(endpoint))
```

Comment:
How shall we handle this for development?

Reply:
Hmm if we run and build it locally, we can actually configure our .env locally to accept the the PR endpoints. 

If we use the Github actions bot's channel, we'll have to provision for a domain for the PR haha. I saw a package previously that could add domain creation into serverless.yml. But I'm not sure if we should use the package though :/

What do you think?(:

Reply:
https://github.com/amplify-education/serverless-domain-manager

Reply:
Actually I was thinking about it, I feel that the library feels too buggy to be used...

Should I just look into other ways of automating and posting the domain url? :/